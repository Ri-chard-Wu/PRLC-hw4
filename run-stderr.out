calc_merkle_root() time: 10886 us
nonceSearch() time: 12962281 us
