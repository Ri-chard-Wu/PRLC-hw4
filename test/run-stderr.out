k[0]: 456
k[0]: 456
k[0]: 456
k[0]: 456
k[0]: 456
k[0]: 456
k[0]: 456
k[0]: 456
k[0]: 456
k[0]: 456
